### ISSUE TITLE
`npm install purecss` Fails on windows 10

### ISSUE THREAD

**Issue by ghost** [pure-css/pure#647]
### Description
`npm install purecss` Fails on windows 10

I tried open cmd as adminastrator and change permissions for node_module folder,but still fails...

### Log
```
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules\chokidar\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.1: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"ia32"})
npm WARN angular-for-wordpress@1.0.0 No repository field.
npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "purecss"
npm ERR! node v7.2.0
npm ERR! npm  v3.10.9
npm ERR! path c:\xampp\htdocs\Angular\node_modules\.staging\purecss-3eef8c0b
npm ERR! code EPERM
npm ERR! errno -4048
npm ERR! syscall rename

npm ERR! Error
npm ERR!     at destStatted (C:\Program Files\nodejs\node_modules\npm\lib\install\action\finalize.js:25:7)
npm ERR!     at C:\Program Files\nodejs\node_modules\npm\node_modules\graceful-fs\polyfills.js:264:29
npm ERR!     at FSReqWrap.oncomplete (fs.js:111:15)
npm ERR!
npm ERR! Error: EPERM: operation not permitted, rename 'c:\xampp\htdocs\Angular\node_modules\.staging\purecss-3eef8c0b' -> 'c:\xampp\htdocs\Angular\node_modules\purecss'
npm ERR!  { Error
npm ERR!     at destStatted (C:\Program Files\nodejs\node_modules\npm\lib\install\action\finalize.js:25:7)
npm ERR!     at C:\Program Files\nodejs\node_modules\npm\node_modules\graceful-fs\polyfills.js:264:29
npm ERR!     at FSReqWrap.oncomplete (fs.js:111:15)
npm ERR!
npm ERR! Error: EPERM: operation not permitted, rename 'c:\xampp\htdocs\Angular\node_modules\.staging\purecss-3eef8c0b' -> 'c:\xampp\htdocs\Angular\node_modules\purecss' parent: 'angular-for-wordpress' }
npm ERR!
npm ERR! Please try running this command again as root/Administrator.
```

**Comment by redonkulus** [comment]
Does this happen with any other packages? I do not have Windows so its hard for me to test. 

If anyone else with Windows could test this out, that would help.

**Comment by zeroic1** [comment]
no,it does not happens with any other packages.

**Comment by redonkulus** [comment]
Looks like this is related to graceful-fs library? I'm not sure there is much that can be done here.

**Comment by Calinou** [comment]
I have tested this on Windows 10 with Node.js 9.1.0, and cannot reproduce this issue with npm 5.5.1 and Yarn 1.3.2. (I used `npm install --save purecss` / `yarn add purecss` to test this.)

**Comment by redonkulus** [comment]
@calinou thanks for testing. can you try on node 6 and 8 as well?

**Comment by Calinou** [comment]
I just tested it on Node.js 6.12.0 and 8.9.1, and it works.

**Comment by billti** [comment]
I just tried this and got the same failure. Updating Node.js didn't help.

I narrowed it down in your devDependencies to `"grunt-contrib-compress": "^1.4.0"`. Looking at their issues it appears to be the same as described in https://github.com/gruntjs/grunt-contrib-compress/issues/190 . This fixed shortly after 1.4.0, and indeed updating that dependency to 1.4.3 resolves the issue for me.
